Smooth Line View/SmoothLineView.m

```diff
@@ -265,6 +265,8 @@ - (UIBezierPath*) path
 
 - (void) setPath:(UIBezierPath*) bezierPath
 {
+    CGMutablePathRef oldPath = _path;
+    CGPathRelease(oldPath);
```

Comment:
Perhaps `NSAssert(bezierPath, ...)` to make sure ... I don't know what would happen with `CGPathCreateMutableCopy(bezierPath.CGPath)`